portfolio/src/main/webapp/passions.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+    <title>Portofolio &mdash; Andrei Topala</title>
+
+    <!-- Bootstrap open-source CSS framework -->
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
+    <link rel="stylesheet" href="style.css">
+    <script src="https://use.fontawesome.com/releases/v5.13.0/js/all.js" crossorigin="anonymous"></script>
+    <script src="script.js"></script>
+    </head>
+  <body>
+    <!-- Navigation Bar -->
+    <nav class="navbar navbar-expand-lg navbar-dark fixed-top">
+      <div class="container">
+        <a class="navbar-brand" href="index.html">Home</a>
+
+        <!-- Hamburger Menu on small resolutions -->
+        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="navbarResponsive" aria-controls="navbarResponsive" aria-expanded="false" aria-label="Toggle navigation">
```

Comment:
There isn't an enforced rule for line length in the style guide for plain HTML, but some groups may have an internal rule for improving readability. You may want to cut the line at 80/100/120 characters length.
In case, please remember line-wrapping requests 4 spaces indentation.
https://google.github.io/styleguide/htmlcssguide.html#HTML_Line-Wrapping